docs/ApplyConfig.md

```diff
@@ -69,7 +69,21 @@ If you would rather use the filevault-package-maven-plugin for building the pack
 </plugin>
 ```
 
-The `*.yaml` files are installed directly from the package content and respect the [run mode semantics](Configuration.md). Otherwise there is no limitation, so the YAML files will be picked up from anywhere in the package (as long as the parent node does not contain a `.` followed by one or multiple not matching run modes).
+The `*.yaml` files are installed directly from the package content and respect the [run mode semantics](Configuration.md). Otherwise there is no limitation, so the YAML files will be picked up from anywhere in the package (as long as the parent node does not contain a `.` followed by one or multiple not matching run modes). If you wish to limit where the YAML files are looked upon you can add one or more package properties with a regular expression to match the desired paths. The properties names should start with `actool.hook.paths.patterns`:
+
+```
+<plugin>
+    <groupId>org.apache.jackrabbit</groupId>
+    <artifactId>filevault-package-maven-plugin</artifactId>
+    <configuration>
+        <properties>
+            <installhook.actool.class>biz.netcentric.cq.tools.actool.installhook.AcToolInstallHook</installhook.actool.class>
+            <actool.hook.paths.patterns.groups>/apps/myapp/acl/groups.*</actool.hook.paths.patterns.groups>
```

Comment:
You should clarify whether this is the repository path or the content-package files names (within the ZIP). Those may be different: https://jackrabbit.apache.org/filevault/vaultfs.html#Filename_escaping

Reply:
if it is file name format, you should explicitly state that the `jcr_root` is automatically prefixed....

Reply:
I stopped adding the /jcr_root automatically and adjusted the documentation. users need to provide a regex that matches the full path within the archive.

Reply:
here is the updated doc https://github.com/Netcentric/accesscontroltool/blob/33cfac6adcfb0e7e393efd3f071b9c5bec9a1a97/docs/ApplyConfig.md#installation-hook